common/theme.c

```diff
@@ -264,3 +266,32 @@ void load_theme(struct theme_config *theme, struct mux_config *config, struct mu
 
     mini_free(muos_theme);
 }
+
```

Comment:
I added the new methods here as I thought it made the most sense since we are applying theme styling.  This way the same methods are used for all screens